prow/plugins/trigger/push.go

```diff
@@ -17,12 +17,31 @@ limitations under the License.
 package trigger
 
 import (
+	"fmt"
+
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/plank"
 )
 
+func notifyOnSlackIfManualMerge(sc slackClient, pe github.PushEvent) error {
+	if sc == nil {
+		return nil
+	}
+	if pe.Pusher.Name != "k8s-merge-robot" && pe.Sender.Login != "k8s-merge-robot" {
+		message := fmt.Sprintf("Warning: %s manually merged %s", pe.Pusher.Name, pe.Compare)
+		if err := sc.WriteMessage(message, "sig-contribex"); err != nil {
+			return err
+		}
+		return sc.WriteMessage(message, "kubernetes-dev")
+	}
+	return nil
+}
+
 func handlePE(c client, pe github.PushEvent) error {
+	if err := notifyOnSlackIfManualMerge(c.SlackClient, pe); err != nil {
```

Comment:
I created a "slackevents" plugin, added push handler. 
I also tested with config.yaml (also trivial change to plugins.yaml) with
//all these repos sent message to these slack channels if pusher is NOT in the whitelist
slackevents:
- repos:
  - kubernetes
  channels: 
  - kubernetes-dev
  - sig-contribex


I want to add another test  /prow/cmd/hook/events_test.go that runs server.handlePushEvent(pe). I just need to figure out where to add/test config_test.yaml and plugins_test.yaml.